packages/client/src/components/SafeContacts.js

```diff
@@ -1,8 +1,7 @@
 import React, { useState } from "react";
 import { isEmpty } from "lodash";
-import InputAddress from "./InputAddress";
-import InputText from "./InputText";
```

Comment:
Why did we delete this?

Reply:
InputAddress is moved to library/InputAddress, 
InputText is only used on this page and it performs exactly like a normal text input, no reason to create a separate file for it. On the other hand, if we want to normalize the input behaviors in the future, we can create inputText reusable component and apply it across the board

Reply:
It would be useful to create a new component or modify the one that is deleted so it can also accept things like:
- label
- flag - if the field is required (to show the *)
- placeholder
- onChange
That way we will unify the look and have less code written.

Reply:
100% agree. Would you mind creating a ticket for this?

Reply:
https://linear.app/dappercollectives/issue/VSL-202